tools/gen_compdb.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
```

Comment:
Consider naming this file gen_compilation_database.py, just for consistency with Envoy.

Reply:
SGTM.